pyspark/bigdl/keras1/converter.py

```diff
@@ -1099,11 +1383,11 @@ def to_bigdl_init(self, kinit_method): # kinit_method is a string
 
     def to_bigdl_reg(self, reg):  # reg is a dict
         if reg:
-           raise Exception("will support reg very soon")
+            return BRegularizer(reg['l1'], reg['l2'])
```

Comment:
what if `l2` isn't in the dict?

Reply:
For the `reg` dict, if `l2` doesn't exist, it is `0`. In other words, no matter what regularizer passed in to a keras layer (l1, l2 or l1l2), in the dictionary, it will always have {l1: x; l2:y}.